core/src/apps/webauthn/credential.py

```diff
@@ -44,6 +51,20 @@
 _U2F_KEY_PATH = const(0x80553246)
 
 
+def truncate_utf8(string: str, max_bytes: int) -> str:
```

Comment:
Maybe we can move this into `src/trezor/utils.py` or some other place where we can use

Reply:
Done.